config/environments/production.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.configure do
+  # Prepare the ingress controller used to receive mail
+
+  # Set this to appropriate ingress service for which the options are :
+  # :relay for Exim, Postfix, Qmail
+  # :mailgun for Mailgun
+  # :mandrill for Mandrill
+  # :postmark for Postmark
+  # :sendgrid for Sendgrid
+  config.action_mailbox.ingress = :sendgrid
```

Comment:
this option can be an environment variable

Reply:
I have created another issue to address this: https://github.com/chatwoot/chatwoot/issues/777